### PR TITLE
Outputted HTML file will now open automatically

### PR DESCRIPTION
**Pull request by taahamahdi**
Renamed output HTML file to index.html

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -40,6 +40,8 @@ def call
         puts "Prettified message files saved: output/prettified/messages"
         puts final_output[:output_strings].join("\n")
         puts "Done!"
+
+        Utils::open_html_graphs()
```
Comment (Brainicism) [inline app.rb]: The `()` is not required for argumentless method calls

File: lib/utils.rb [hunk 2]
```diff
@@ -67,5 +68,33 @@ def write_output_csv(output, directory, type)
...
+            def OS.unix?
+                !OS.windows?
+            end
+
+            def OS.linux?
+                OS.unix? and not OS.mac?
+            end
+        end
+
+        def open_html_graphs()
```
Comment (Brainicism) [inline lib/utils.rb]: Same comment as above.
Also, does this need to be in the module?

File: app.rb (hunk 1 above)
Comment (taahamahdi) [reply app.rb]: You're right, sorry about that!

File: lib/utils.rb (hunk 2 above)
Comment (taahamahdi) [reply lib/utils.rb]: I don't think it's in the module, but I'll move it into another file so it doesn't mess with readability :]
I'll commit the corrections in a bit! Thanks for the help!

**Pull request merged**